irods/version.py

```diff
@@ -1 +1 @@
-__version__ = '0.8.4'
+__version__ = '0.8.5'
```

Comment:
New version.

Reply:
yes, this will get released quickly once merged.

Reply:
please split this to a separate commit

Reply:
Sure